libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -16,12 +16,12 @@ import {
   useFeature,
   ClusterCreateParams,
   getSupportedCpuArchitectures,
+  PlatformType,
```

Comment:
(Not for this line).
In `ClusterWizardFooter`, we have the `ClusterPlatformIntegrationHint`.
This is for showing the user the banner about the platform integration potential. Since they will select the platform beforehand, do we still want this shown?

Reply:
@nirfarkas WDYT?
It's this banner that appears in host discovery step:
![image](https://github.com/openshift-assisted/assisted-installer-ui/assets/11390125/05fe300e-0318-4b74-889f-9da2713bc4f6)

Reply:
After discuss with Nir, we don't need this banner any more.